Dockerfile

```diff
@@ -1,16 +1,20 @@
 FROM ubuntu:16.04
 
+ENV IMPLY_VERSION 2.2.3
```

Comment:
actually we can use ARG IMPLY_VERSION here, then pass it to the build command:
https://docs.docker.com/engine/reference/builder/#arg

that way when we want to update we just need to update the circleci file.